platforms/access-bridge/src/server.ts

```diff
@@ -50,6 +52,11 @@ export class Server {
     // Register custom endpoints
     registerEndpoints(this.app);
 
+    // The error handler must be registered before any other error middleware and after all controllers
+    if (Sentry.getClient()) {
```

Comment:
Suggest we move this to logger.ts and wrap it in a `logger.init()` method or something.